packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -287,6 +289,18 @@ export class IndexedDbPersistence implements Persistence {
           }
 
           if (!currentPrimary.allowTabSynchronization) {
+            // Fail the `canActAsPrimary` check if the current leaseholder has
+            // not opted into multi-tab synchronization. If this happens at
+            // client startup, we reject the Promise returned by
+            // `enablePersistence()` and the user can continue to use Firestore
+            // with in-memory persistence.
+            // If this fails during a lease refresh, we will instead block the
+            // AsyncQueue from execution further operations. Note that this is
+            // acceptable since mixing & matching different `synchronizeTabs`
+            // settings is not officially supported.
```

Comment:
"not officially supported" makes it sound like there's some amount of unofficial support, but in reality it would be completely broken, right?

Reply:
I removed `officially`. It's probably going to be very very broken.